### DISCUSSION TITLE
TCA and non Darwin platforms.

### DISCUSSION THREAD

**Discussion by 3a4oT** [pointfreeco/swift-composable-architecture#2953] [Ideas]
Yesterday, I read a blog post on swift.org - [Writing GNOME Apps with Swift](https://www.swift.org/blog/adwaita-swift/). So, as you already guessed, my question - is it possible to have TCA on Linux or Windows? :)

**Comment by mbrandonw** [comment]
Hi @3a4oT, it's definitely a goal of ours with TCA, but it will take time to get there as it will need to introduce breaking changes to the library. And so it will need to be a 2.0 or possibly 3.0 consideration.
